### PR TITLE
Remove Deprecated CPP Funds Mixer Configuration

### PR DESCRIPTION
**Pull request by dmigwi**
The `CSPPServer` and `Cert` configurations are no longer necessary when setting the mixing configuration.
To activate a the Peer-to-peer of a boolean true that is required.

### CODE REVIEW

File: client/webserver/site/src/js/wallets.ts [hunk 1]
```diff
@@ -1521,7 +1522,7 @@ export default class WalletsPage extends BasePage {
     const { page, selectedAssetID: assetID, mixerStatus } = this
     if (!mixerStatus) return
     Doc.hide(page.mixConfigErr)
-    const cfg = { assetID: assetID } as any
+    const cfg = { assetID: assetID, IsMixingEnabled: page.enableMixing } as any
```
Comment (ukane-philemon) [inline client/webserver/site/src/js/wallets.ts:1525]: IsMixingEnabled -> isMixingEnabled

File: client/webserver/site/src/js/wallets.ts [hunk 2]
```diff
@@ -1521,7 +1522,7 @@ export default class WalletsPage extends BasePage {
...
     const loaded = app().loading(page.mixerConfigForm)
     if (Doc.isDisplayed(page.mixAddrInputBox)) {
       cfg.serverAddr = page.mixAddrInput.value
```
Comment (ukane-philemon) [inline client/webserver/site/src/js/wallets.ts:1528]: Do users still need to provide mixing server addr? Can remove form elements that are no longer required.

**Review by ukane-philemon — CHANGES_REQUESTED** [review]

File: client/webserver/site/src/js/wallets.ts (hunk 2 above)
Comment (dmigwi) [reply client/webserver/site/src/js/wallets.ts:1528]: No they don't It is no longer required. I will remove it.

File: client/webserver/site/src/js/wallets.ts (hunk 1 above)
Comment (dmigwi) [reply client/webserver/site/src/js/wallets.ts]: Resolved!

File: client/webserver/site/src/js/wallets.ts (hunk 2 above)
Comment (ukane-philemon) [reply client/webserver/site/src/js/wallets.ts]: Okay.

**Review by buck54321 — CHANGES_REQUESTED** [review]
You added a commit while I was reviewing, but these are the rest of the needed changes.
https://github.com/buck54321/dcrdex/compare/0a109e1...buck54321:dcrdex:newmixing

**Comment by dmigwi** [comment]
> You added a commit while I was reviewing, but these are the rest of the needed changes. [buck54321/dcrdex@0a109e1...buck54321:dcrdex:newmixing](https://github.com/buck54321/dcrdex/compare/0a109e1...buck54321:dcrdex:newmixing)

@buck54321 your changes are way better. Please let us close the PR in favour of your changes!

File: client/asset/dcr/spv.go [hunk 3]
```diff
@@ -1059,7 +1059,7 @@ func (w *spvWallet) PurchaseTickets(ctx context.Context, n int, vspHost, vspPubK
 		VSPFeePercent:        vspClient.FeePercentage,
 	}
 
-	if mixCfg.enabled {
+	if isMixingEnabled {
 		accts := w.Accounts()
 		mixedAccountNum, err := w.AccountNumber(ctx, accts.PrimaryAccount)
```
[the remark this answers is not in the document]
Comment (dmigwi) [reply client/asset/dcr/spv.go]: I don't see then need of setting the `Mixed` parameter to the `req` struct as it is not used past here. If you think we should follow that route let me know and I will make the change.

File: client/asset/dcr/spv.go [hunk 4]
```diff
@@ -1047,7 +1047,7 @@ func (w *spvWallet) rescan(ctx context.Context, fromHeight int32, c chan wallet.
 
 // PurchaseTickets purchases n tickets, tells the provided vspd to monitor the
 // ticket, and pays the vsp fee.
-func (w *spvWallet) PurchaseTickets(ctx context.Context, n int, vspHost, vspPubKey string, mixCfg *mixingConfig) ([]*asset.Ticket, error) {
+func (w *spvWallet) PurchaseTickets(ctx context.Context, n int, vspHost, vspPubKey string, isMixingEnabled bool) ([]*asset.Ticket, error) {
```
Comment (dmigwi) [inline client/asset/dcr/spv.go]: This `isMixingEnabled` parameter ought to be the same one managed by the Native wallet instance. At least that is my understanding.

File: client/asset/dcr/spv.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (dmigwi) [reply client/asset/dcr/spv.go]: I think how you implemented it is okay

[the remark this answers is not in the document]
Comment (buck54321) [reply client/asset/dcr/spv.go]: I thought I had deleted the comment. My bad.

**Review by buck54321 — APPROVED** [review]

**Pull request merged**